doc/index.rst

```diff
@@ -1,3 +1,5 @@
+:og:description: Catalyst is an experimental package that enables just-in-time (JIT) compilation of PennyLane programs. Compile the entire quantum-classical workflow.
```

Comment:
**Question:** Isn't this description already captured in `conf.py`?

Reply:
Nope, the one in `conf.py` doesn't seem to do anything :( This is the one that is properly being registered for the docs homepage

Reply:
Actually, the `ogp_description` option in `conf.py` doesn't even seem to appear in the README of the extension.

Perhaps we should remove it?

Reply:
let's do it! The extension is really badly documented, I had to look through old issues/other packages that use it like `urllib3` in order to work out how it works :(